packages/runtime/runtime-utils/src/summarizerNode.ts

```diff
@@ -301,49 +346,27 @@ export class SummarizerNode implements ISummarizerNode {
             }
         }
 
+        if (simple) {
+            const result = this.simpleSummary();
+            assert(result !== false, "Cannot generate simple summary");
```

Comment:
If that's the intention, I'd change simplerSummary to not return false, but rather throw.

Reply:
True, it does throw in all cases. I'll parameterize the error object.